lib/gollum/views/helpers.rb

```diff
@@ -51,6 +51,13 @@ def self.included(base)
         def rocticon(symbol, parameters = {})
           Octicons::Octicon.new(symbol, parameters).to_svg
         end
+
+        def rocticon_css(symbol, parameters = {})
```

Comment:
I dont' know if it's faster per se, but here's a more Ruby-ish way of achieving what you want:
```ruby
def rocticon_css(symbol, parameters = {})
  octicon = Octicons::Octicon.new(symbol, parameters.merge({xmlns: 'http://www.w3.org/2000/svg'}))
  [:width, :height].each {|option| octicon.options.delete(option)}
  octicon.to_svg
end
```
The outputted string from this snippet is already utf-8 encoded.

Reply:
Did you look at the resulting HTML for that snippet? Looking at the source for `Octicon` it seemed to me that deleting the `:width` and `:height` options would result in the calculation of a default width and height. (We need the SVG not to set any height/width at all, for use in backgrounds.)

Reply:
```irb
jruby-9.1.13.0 :001 > require 'octicons'
 => true 
jruby-9.1.13.0 :002 > octicon = Octicons::Octicon.new('x', {}.merge({xmlns: 'http://www.w3.org/2000/svg'}))
 => #<Octicons::Octicon:0x341814d3 @symbol="x", @keywords=["remove", "close", "delete"], @height=16, @width=12, @options={:xmlns=>"http://www.w3.org/2000/svg", :class=>"octicon octicon-x", :viewBox=>"0 0 12 16", :version=>"1.1", :width=>12, :height=>16, :"aria-hidden"=>"true"}, @path="<path fill-rule=\"evenodd\" d=\"M7.48 8l3.75 3.75-1.48 1.48L6 9.48l-3.75 3.75-1.48-1.48L4.52 8 .77 4.25l1.48-1.48L6 6.52l3.75-3.75 1.48 1.48L7.48 8z\"/>"> 
jruby-9.1.13.0 :003 > [:width, :height].each {|option| octicon.options.delete(option)}
 => [:width, :height] 
jruby-9.1.13.0 :004 > octicon.to_svg
 => "<svg xmlns=\"http://www.w3.org/2000/svg\" class=\"octicon octicon-x\" viewBox=\"0 0 12 16\" version=\"1.1\" aria-hidden=\"true\"><path fill-rule=\"evenodd\" d=\"M7.48 8l3.75 3.75-1.48 1.48L6 9.48l-3.75 3.75-1.48-1.48L4.52 8 .77 4.25l1.48-1.48L6 6.52l3.75-3.75 1.48 1.48L7.48 8z\"/></svg>" 
jruby-9.1.13.0 :005 > 
```

Reply:
No height or width set; `to_svg` just processes the keys present in the options hash.

Reply:
Sweeet! Thanks